git-server/src/lib.rs

```diff
@@ -106,11 +112,52 @@ impl Context {
 
         Ok(())
     }
+
+    /// Grabs projects names and uses projects' names as alias for urns
+    fn grab_aliases(&self) -> Result<(), Error> {
```

Comment:
Can we call this something more descriptive, like `populate_aliases` or `load_aliases`?

Reply:
Fixed https://github.com/radicle-dev/radicle-client-services/pull/74/commits/67c4936ad316ef7490d2c15d4b88ac055b09ffe9#diff-b672a15ad14f54528cebd77aee76d3007962d44bda987b3e0a34597ec80fd6b1R116-R117